diff --git a/node_modules/react-native-native-video-player/android/build.gradle b/node_modules/react-native-native-video-player/android/build.gradle
index c7cfff8..1a1ea9c 100644
--- a/node_modules/react-native-native-video-player/android/build.gradle
+++ b/node_modules/react-native-native-video-player/android/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.3"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 26
         versionCode 1
         versionName "1.0"
     }
@@ -21,6 +21,6 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.0.1'
+    compile 'com.android.support:appcompat-v7:26.1.0'
     compile 'com.facebook.react:react-native:+'
 }
diff --git a/node_modules/react-native-native-video-player/android/src/main/java/com/wog/videoplayer/VideoPlayerPackage.java b/node_modules/react-native-native-video-player/android/src/main/java/com/wog/videoplayer/VideoPlayerPackage.java
index e742d90..771abb5 100644
--- a/node_modules/react-native-native-video-player/android/src/main/java/com/wog/videoplayer/VideoPlayerPackage.java
+++ b/node_modules/react-native-native-video-player/android/src/main/java/com/wog/videoplayer/VideoPlayerPackage.java
@@ -20,7 +20,7 @@ public class VideoPlayerPackage implements ReactPackage {
         return Arrays.<NativeModule>asList(new VideoPlayerModule(reactContext));
     }
 
-    @Override
+    // @Override // remove this if RN > 0.45.0
     public List<Class<? extends JavaScriptModule>> createJSModules() {
         return Collections.emptyList();
     }
